src/components/Sum/Sum.jsx

```diff
@@ -1 +1,15 @@
-// export const Sum = () => ();
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>
+    Sum of
+    {' '}
+    { a }
+    {' '}
+    and
+    {' '}
+    { b }
+    {' '}
+    is
+    {' '}
+    {a + b }
+  </p>
```

Comment:
![image](https://github.com/mate-academy/react_sum/assets/59209337/ba2b8e3c-71d8-4888-ac61-4f2205d04ddb)

Reply:
The problem is that when I run 'npm run lint' or 'git commit - m' it automatically change my code into solution that u consider as wrong.